code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -160,7 +160,7 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 	/// For example, if `low_pop_maximum_threat` is 50, `low_pop_player_threshold` is 20,
 	/// and the number of readied players is 10, then the highest threat that can roll is
 	/// lerp(50, 100, 10 / 20), AKA 75.
-	var/low_pop_maximum_threat = 40
+	var/low_pop_maximum_threat = 25
```

Comment:
Note that the number of roundstart ready players, which this value is based around, is around half of the number of players in the round at minute 10 due to a lot of latejoiners. i. e. these values run off of unrealistically low pop counts.

Reply:
The value is LERP'd